lib/html-tags.js

```diff
@@ -49,13 +51,13 @@ function htmlTagObjectToString (tagDefinition, xhtml) {
  * @param {string} tagName
  * the name of the tag e.g. 'div'
  *
- * @param {{[attributeName: string]: string|boolean}} [attributes]
+ * @param {Object.<string, string | boolean>} [attributes]
  * tag attributes e.g. `{ 'class': 'example', disabled: true }`
  *
  * @param {string} [innerHTML]
  *
- * @param {{[attributeName: string]: string|boolean}} [meta]
- * meta information about the tag e.g. `{ 'pluhin': 'html-webpack-plugin' }`
+ * @param {Object.<string, string | boolean>} [meta]
```

Comment:
Also fixed the JSDoc definitions for `object` entry

Reply:
Can we please keep`attributeName` and add `undefined` & `null`?

Reply:
Of course, do you want to restore both changes on the JSDoc?

Reply:
Yes and please also for `typings.d.ts` :)

Reply:
The change on `typingins.d.ts` are necessary to reflect the change

Reply:
You are right :) sorry I didn't see `typings.d.ts` - here it should be the same as in `typings.d.ts`: 

```suggestion
 * @param {{[attributeName: string]: string | boolean | null | undefined}} [meta]
```